zero/gkyl_rect_grid_priv.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <gkyl_rect_grid.h>
+
+/* Checks if given point is in given cell
```

Comment:
@jRoeltgen please revise these comments, I think they are not right for in_dir

Reply:
Changed "Checks if given point is in given cell" to "Find upper and lower boundaries of given cell"